### PR TITLE
=tes #16074 remove return statements from scala sources

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

File: akka-testkit/src/test/scala/akka/testkit/Coroner.scala [hunk 1]
```diff
@@ -225,7 +225,7 @@ object Coroner {
         for (li ← locks) appendMsg("\t- ", li)
       }
       sb.append('\n')
-      return sb.toString
+      sb.toString
```
Comment (patriknw) [inline akka-testkit/src/test/scala/akka/testkit/Coroner.scala]: wow :)

**Comment by patriknw** [comment]
LGTM

**Comment by johanandren** [comment]
PLS BUILD

**Comment by johanandren** [comment]
iur

**Comment by johanandren** [comment]
PLS BUILD

**Comment by ktoso** [comment]
Nice :-) Are we down to 0 return uses yet? :-)

**Comment by johanandren** [comment]
Yes indeed

**Comment by patriknw** [comment]
Refs #16074

**Pull request merged**